ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/image/ImageComponentView.kt

```diff
@@ -210,6 +213,54 @@ private fun ImageComponentView_Preview(
     }
 }
 
+@Preview
+@Composable
+private fun ImageComponentView_Bigger_Container_Fill_Fit_FitModeFill_Preview() {
+    val themeImageUrls = previewThemeImageUrls(widthPx = 100u, heightPx = 100u)
+    Box(
+        modifier = Modifier
+            .background(ComposeColor.Red)
+            .fillMaxWidth()
+            .height(200.dp)
```

Comment:
Maybe these can be "swapped", we can more easily see the `Fit` behavior?
```suggestion
            .fillMaxHeight()
            .width(200.dp)
```

Reply:
Whoops needed an import 😅

Reply:
Haha sorry! 😳